src/hooks/auth0-provider.tsx

```diff
@@ -91,7 +103,7 @@ const Auth0Provider = ({domain, clientId, children}) => {
   );
 
   const clearSession = useCallback(
-    async (...options) => {
+    async (...options: any[]) => {
```

Comment:
Would it make sense to provide a typed options parameter here instead of resorting to any?